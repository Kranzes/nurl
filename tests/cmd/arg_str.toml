args = [
    "https://github.com/nix-community/nurl",
    "v0.3.0",
    "--arg-str",
    "name",
    "nurl-src",
]
