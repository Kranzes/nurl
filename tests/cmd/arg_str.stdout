fetchFromGitHub {
  owner = "nix-community";
  repo = "nurl";
  rev = "v0.3.0";
  hash = "sha256-jZ+cCp1THDhfHH5yMmRPjGuthOqsgcF/3OjZ61FMdA4=";
  name = "nurl-src";
}